crates/store/re_types/definitions/rerun/components/graph_edge.fbs

```diff
@@ -0,0 +1,19 @@
+namespace rerun.components;
+
+// ---
+
+/// An edge in a graph connecting two nodes.
+///
+/// Depending on the context this could represent a directed or undirected edge.
+//
+// TODO(grtlr): Why do we need `Default` here?
+struct GraphEdge (
+  "attr.arrow.transparent",
+  "attr.python.aliases": "Tuple(int, int)",
+  "attr.rust.derive": "Copy, Default, PartialEq, Eq, PartialOrd, Ord, Hash, bytemuck::Pod, bytemuck::Zeroable",
+  "attr.rust.repr": "transparent",
+  "attr.rust.custom_clause":
+    'cfg_attr(feature = "serde", derive(::serde::Serialize, ::serde::Deserialize))'
+) {
+  edge: [rerun.datatypes.GraphNodeId: 2] (order: 100);
```

Comment:
Is there any practical use (in robotics, primarily) in support hypergraphs (where an edge can join multiple nodes)?

Reply:
Hmm 🤔, I'm not aware of any practical applications that use hypergraphs. There also does not seem to be any support in the [`petgraph` crate](https://docs.rs/crate/petgraph/latest), so maybe we should focus on regular graphs first. If we hypergraphs will become a thing, we can probably add a `HyperGraphEdge` component to remain backwards compatible.

Of course, an hypergraph could also be "normalized" by adding virtual nodes for the edges that join multiple nodes.